components/fxa-client/src/http_client.rs

```diff
@@ -219,6 +224,19 @@ impl FxAClient for Client {
         Ok(Self::make_request(request)?.json()?)
     }
 
+    fn oauth_introspection_with_refresh_token(
```

Comment:
naming nit: I think this should be called something like `oauth_introspect_refresh_token`, since the token is the thing that you're getting info about, rather than just a thing that enables you to take the action in question.